scripts/install.sh

```
@@ -1,14 +1,23 @@
+## DOCKER_POOL_FORCE_OVERRIDE - Force override Docker address pool configuration (default: false)
+## AUTOUPDATE - Set to "false" to disable auto-updates
+
 set -e # Exit immediately if a command exits with a non-zero status
 ## $1 could be empty, so we need to disable this check
 #set -u # Treat unset variables as an error and exit
 set -o pipefail # Cause a pipeline to return the status of the last command that exited with a non-zero status
 CDN="https://cdn.coollabs.io/coolify"
 DATE=$(date +"%Y%m%d-%H%M%S")
 
-VERSION="1.7"
+VERSION="1.8"
```

Shellcheck (0.10.0)

[warning] VERSION appears unused. Verify use (or export if used externally).

(SC2034)